client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -424,7 +518,10 @@ class DomainDetailsForm extends Component {
 			} );
 
 		let title;
-		if ( needsOnlyGoogleAppsDetails ) {
+		// TODO: gather up tld specific stuff
+		if ( this.state.currentStep === 'fr' ) {
+			title = this.props.translate( '.FR Registration' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **66** times:
`translate( 'Registration' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tld/fr-extra-fields-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).